src/test/kotlin/org/openrewrite/java/spring/NoAutowiredTest.kt

```diff
@@ -61,7 +56,6 @@ class NoAutowiredTest : JavaRecipeTest {
         """
     )
 
-    @Disabled
     @Issue("https://github.com/openrewrite/rewrite-spring/issues/78")

```

Comment:
These tests must be more exhaustive. 

have two constructors, one constructor plus auto-wired field and/or setter, etc.